### DISCUSSION TITLE
How to add bg color to the title header of the table using nuxt ui.

### DISCUSSION THREAD

**Discussion by astro-ninja** [nuxt/nuxt#26747] [General]
Hello, Good day everyone , asking for help I'm new to nuxt ui, I would like to ask how to add bg color to the tittle header of the table using nuxt ui. 

<script setup lang="ts">
const columns = [{
  key: 'id',
  label: 'ID'
}, {
  key: 'name',
  label: 'User name'
}, {
  key: 'title',
  label: 'Job position'
}, {
  key: 'email',
  label: 'Email'
}, {
  key: 'role'
}]

const people = [{
  id: 1,
  name: 'Lindsay Walton',
  title: 'Front-end Developer',
  email: 'lindsay.walton@example.com',
  role: 'Member'
}, {
  id: 2,
  name: 'Courtney Henry',
  title: 'Designer',
  email: 'courtney.henry@example.com',
  role: 'Admin'
}, {
  id: 3,
  name: 'Tom Cook',
  title: 'Director of Product',
  email: 'tom.cook@example.com',
  role: 'Member'
}, {
  id: 4,
  name: 'Whitney Francis',
  title: 'Copywriter',
  email: 'whitney.francis@example.com',
  role: 'Admin'
}, {
  id: 5,
  name: 'Leonard Krasner',
  title: 'Senior Designer',
  email: 'leonard.krasner@example.com',
  role: 'Owner'
}, {
  id: 6,
  name: 'Floyd Miles',
  title: 'Principal Designer',
  email: 'floyd.miles@example.com',
  role: 'Member'
}]
</script>

<template>
  <UTable :columns="columns" :rows="people" />
</template>

![table](https://github.com/nuxt/nuxt/assets/150634086/3857b399-54ce-45e3-9b85-5a49b98c4a9d)

**Comment by Scorpion197** [comment]
@astro-ninja I did it like this however I don't know if it's a good or a bad practice: 
```css
<style scoped>
:deep(thead) {
    background-color: #3d3d3d !important;
}
</style>

```
